### PR TITLE
Build "About" page for site

### LINKED ISSUE #289: Build "About" page for site
Design is in #99

### PR DESCRIPTION
**Pull request by gesa**
Fixes #289

### CODE REVIEW

**Comment by toolness** [comment]
@ScottDowne can you review this one please?

File: less/components/about-page.less [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    margin-bottom: 2.5rem;
+    margin-top: 2.5rem;
+    padding-left: 2.5rem;
+    padding-right: 2.5rem;
+    @media screen and (min-width: @screen-sm) {
+      padding-left: 0;
+    }
+  }
+  p {
+    max-width: 50rem;
```
Comment (ScottDowne) [inline less/components/about-page.less]: This is making the three sub titles in the three icons not be centered.

The text that are problems are 
- Start a conversation on Twitter
- Email us anytime
- Connect on the Discourse forum

**Comment by ScottDowne** [comment]
Double check that the travis failure isn't caused by you.

**Comment by ScottDowne** [comment]
R+

**Pull request merged**